k9mail/src/main/res/values-night/colors.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
 
     <color name="logo_primary_color">@android:color/white</color>
+    <color name="primary_color_90">@color/message_list_selected_background_color_dark</color>
```

Comment:
This association seems rather confusing... why is the primary_color_90 the message list selected...? can we do the association the other way around? since primary_color_90 sounds more general than the other one.

Reply:
Complexity case, as you can see our legacy code is not reusable enough to satisfy those requirements.
I just opened a new one with dedicated colors and selector for the attachment view element.